### PR TITLE
Closes #540: Replace structure block button with custom block button

### LINKED ISSUE #540: Replace structure block button with custom block button
In the CKEditor, the existing Block embed button does not refer to the correct type of blocks we need. It should refer to "content blocks" or use the "custom block" button instead of the existing "block" button.

Task: remove block button and add custom block button to Standard text format CKEditor.

### PR DESCRIPTION
**Pull request by barrettbaffert**
## Description

In the CKEditor, the existing Block embed button does not refer to the correct type of blocks we need. It should refer to "content blocks" or use the "custom block" button instead of the existing "block" button.

Task: remove block button and add custom block button to Standard text format CKEditor.

## Related Issue

https://github.com/az-digital/az_quickstart/issues/540

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

If anyone is using structure blocks within ckeditor on live sites (?) they'd lose the ability to edit them?

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by barrettbaffert** [comment]
Assuming I understand the requirements, I think this is working as expected? 

https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-587.probo.build/test-page

Steps for testing defined below (again, assuming I did it right)

1. start at /admin/structure/block
2. click "Custom block library" tab which takes you to /admin/structure/block/block-content
3. click Add custom block button
4. create the new block and save it
5. create a new page at /node/add/az_flexible_page
6. Use Add Text paragraph type
7. Click the embed button that (when you hover over it) says Embed Content Block
8. search for the block you created in step 4 above
9. save the page and test that it worked

**Review by smayden — APPROVED** [review]
Able to embed block I created on a page following the instructions. Seems to work as expected!

File: modules/custom/az_paragraphs/config/optional/embed.button.az_embed_content_blocks.yml [hunk 1]
```diff
@@ -2,16 +2,16 @@ langcode: en
 status: true
 dependencies:
   module:
-    - block
+    - block_content
     - entity_embed
-label: 'Embed Block'
-id: az_embed_block
+label: 'Embed Content Blocks'
+id: az_embed_content_blocks
```
Comment (barrettbaffert) [inline modules/custom/az_paragraphs/config/optional/embed.button.az_embed_content_blocks.yml:8]:
```suggestion
id: az_embed_content_block
```

File: modules/custom/az_paragraphs/config/optional/embed.button.az_embed_content_blocks.yml [hunk 2]
```diff
@@ -2,16 +2,16 @@ langcode: en
 status: true
 dependencies:
   module:
-    - block
+    - block_content
     - entity_embed
-label: 'Embed Block'
-id: az_embed_block
+label: 'Embed Content Blocks'
```
Comment (barrettbaffert) [inline modules/custom/az_paragraphs/config/optional/embed.button.az_embed_content_blocks.yml:7]:
```suggestion
label: 'Embed Content Block'
```

File: modules/custom/az_paragraphs/config/optional/editor.editor.az_standard.yml [hunk 3]
```diff
@@ -56,9 +56,9 @@ settings:
         -
           name: Embed
           items:
-            - az_embed_content
             - DrupalMediaLibrary
-            - az_embed_block
+            - az_embed_content
+            - az_embed_content_blocks
```
Comment (barrettbaffert) [inline modules/custom/az_paragraphs/config/optional/editor.editor.az_standard.yml:61]:
```suggestion
            - az_embed_content_block
```

**Review by trackleft — APPROVED** [review]

**Comment by barrettbaffert** [comment]
Tested on probo now that all the ID, label, and filename instances of "blocks" have been changed to "block." I believe this is ready to merge now.

https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-587.probo.build/adsfadsdfdsaf

**Review by smayden — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**